tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -17,8 +17,16 @@ Documentation  Test 5-25 - OPS-User-Grant
 Resource  ../../resources/Util.robot
 Suite Setup  Wait Until Keyword Succeeds  10x  10m  Ops User Create
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+Test Teardown  Run Keyword And Ignore Error  Gather VC Logs
 
 *** Keywords ***
+
+Gather VC Logs
+    Log To Console  Collecting VC logs ..
+    Run Keyword And Ignore Error  Gather Logs From ESX Server
+    Run Keyword And Ignore Error  Run  mv vcsupport*.tgz vcsupport.tgz.log
```

Comment:
Don't just move this to a different extension, please zip the file into a zip archive.

Reply:
@mhagen-vmware can we add .tgz as a recognized archive type? Seems a little ott to have a `.tar.gz.zip`